roles/check-swlib/tasks/main.yml

```diff
@@ -0,0 +1,123 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: check-swlib | Define Oracle release
+  #if release set to latest then get most recent release from gi_patches array
+  #if nothing returned then set to "base" in next task
+  set_fact:
+    oracle_rel: "{% if not ansible_loop.first %}{% if item is version(oracle_rel, '>=') %}{{ item }}{% else %}{{ oracle_rel }}{% endif %}{% else %}{{ item }}{% endif %}"
+  loop: "{{ gi_patches | selectattr('base', 'match', oracle_ver) | map(attribute='release') | list }}"
```

Comment:
Same as comment: 
https://github.com/google/bms-toolkit/pull/81/commits/84f1938d7b6008e51ba336067819cb18669bd9d5#r675305979

Can we use tested `loop` expression for consistency?
If the current one is more robust in catching corner, can the other places that use the same for `oracle_rel` be changed after testing?